src/MongoDB/CursorId.c

```diff
@@ -193,24 +222,16 @@ static zend_object* php_phongo_cursorid_create_object(zend_class_entry* class_ty
 
 static HashTable* php_phongo_cursorid_get_debug_info(phongo_compat_object_handler_type* object, int* is_temp) /* {{{ */
 {
-	php_phongo_cursorid_t* intern;
-	zval                   retval = ZVAL_STATIC_INIT;
-
 	*is_temp = 1;
-	intern   = Z_OBJ_CURSORID(PHONGO_COMPAT_GET_OBJ(object));
-
-	array_init(&retval);
-
-#if SIZEOF_ZEND_LONG == 4
-	ADD_ASSOC_INT64_AS_STRING(&retval, "id", intern->id);
-#else
-	ADD_ASSOC_LONG_EX(&retval, "id", intern->id);
-#endif
```

Comment:
We should keep this around so `var_dump()` behavior isn't changed by this PR. That should entail copying this to `php_phongo_cursorid_get_properties_hash` and placing it within a conditional dependent on `is_debug`.